it/distribution_test.py

```diff
@@ -102,6 +101,19 @@ def test_cluster():
     cluster.stop()
 
 
+@it.random_rally_config
+def test_multi_target_hosts(cfg, test_cluster):
+    hosts = '"127.0.0.1:{}"'.format(test_cluster.http_port)
+    target_hosts = '{{ "remote":[{hosts}], "default":[{hosts}] }}'.format(hosts=hosts)
+    client_options = '{{ "default": {{"max_connections": 50}}, "remote": {{"max_connections": 100}} }}'
```

Comment:
`target_hosts` results in invalid JSON. Using F-strings:

```python
hosts = f"127.0.01:{test_cluster.http_port}"
target_hosts = f"{{\"remote\":[\"{hosts}\"], \"default\":[\"{hosts}\"]}}"
client_options = f"{{\"default\": {{\"max_connections\": 50}}, \"remote\": {{\"max_connections\": 100}}}}"

>>> hosts
'127.0.0.1:19200'
>>> target_hosts
'{"remote":["127.0.0.1:19200"], "default":["127.0.0.1:19200"]}'
>>> client_options
'{"default": {"max_connections": 50}, "remote": {"max_connections": 100}}'
```

Reply:
Much cleaner without escapes:

```python
target_hosts = f'{{"remote":["{hosts}"], "default":["{hosts}"]}}'
```

Reply:
I've pushed https://github.com/elastic/rally/pull/1518/commits/379a40b7dfef4f1e2b133583dfd1e7507b044e0c to generate json from dicts again. Thanks for looking.